lib/index.js

```diff
@@ -143,7 +143,7 @@ module.exports = function loader (css, map, meta) {
     return postcss(plugins)
       .process(css, options)
       .then((result) => {
-        result.warnings().forEach((msg) => this.emitWarning(msg.toString()))
+        result.warnings().forEach((msg) => this.emitWarning(new Error(msg.toString())))
```

Comment:
I need to check the Postcss API docs, but I think `warnings` can contain `location` info and a code frame if the plugin author used the appropiated PostCSS API. If that's the case it would be better to reuse the code located in `lib/Error.js` and create a new class `LoaderWarning` (`lib/Warning.js`) based upon that to be consistent in terms of formatting